lib/create-testing-library-rule/detect-testing-library-utils.ts

```diff
@@ -133,6 +133,14 @@ const FIRE_EVENT_NAME = 'fireEvent';
 const CREATE_EVENT_NAME = 'createEvent';
 const USER_EVENT_NAME = 'userEvent';
 const RENDER_NAME = 'render';
+const EXCLUDED_FIND_BY_QUERIES = [
+	//react-test-renderer utils
+	'findByType',
+	'findByProps',
+	'findAll',
+	'findAllByType',
+	'findAllByProps',
```

Comment:
And what if someone uses a custom query with one of these names? We still want them to be reported. no? How can we make sure that we are dealing with a react-test-renderer query?
This relates also to [this comment](https://github.com/testing-library/eslint-plugin-testing-library/issues/673#issuecomment-1281303226).

Reply:
That would be the ideal fix indeed. However, since the utils names seem quite specific and can be chained from built-in queries in React Native, it could be a quick workaround.

I personally don't have the time to implement the ideal fix so we check if queries are coming from `react-test-renderer`, so it's a matter of deciding if we are happy with this workaround (and its trade-offs) or we are only interested in the ideal fix.

Reply:
Another idea could be adding a config for React Native to the plugin, which adds automatically the corresponding query names to the settings to be ignored as custom queries. This would be a workaround too, but perhaps makes more sense.